src/components/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import Link from "next/link";
+import React from "react";
+
+interface ButtonProps {
+  text: string;
+  link: string;
```

Comment:
instead of passing in a link can you pass in an arrow function here. For example it can be `handleSubmit: () => void`. Then this function should be called when the button component is clicked.